tests/test_office_structure/test_get_node.py

```diff
@@ -0,0 +1,32 @@
+import sys
+import os
+import json
+from tests.base import BaseTestCase
+from fixtures.office_structure.get_node_fixtures import (
+    get_node_query,
+    get_node_query_response,
+    get_node_non_existant_id_query,
+    get_node_with_non_existant_id_response
+)
+
+sys.path.append(os.getcwd())
```

Comment:
Great work @ThaDeveloper, Kindly explain what this line is for. If it's not being used. Please remove it

Reply:
@RachaelNantale It automatically injects the path to the current working directory

Reply:
It's not really necessary, I have removed it. Thanks